.gitlab/ci/on-push.yml

```diff
@@ -70,14 +70,15 @@ run-unittests-and-lint:
     - if: '$BUCKET_UPLOAD == "true"'
       when: never
     - if: '$DEMISTO_SDK_NIGHTLY != "true"'
+    - if: '$CI_PIPELINE_SOURCE =~ /^(push|contrib)$/'
```

Comment:
double check this

Reply:
entering in cases of on-push or contribution.